src/programs/project3d/project3d.cpp

```diff
@@ -65,7 +65,8 @@ void Project3DApp::DoInteractiveUserInput()
 	}
 
 	ouput_projection_stack = my_input->GetFilenameFromUser("Output projection stack", "The output image stack, containing the 2D projections", "my_projection_stack.mrc", false);
-	pixel_size = my_input->GetFloatFromUser("Pixel size of images (A)", "Pixel size of input images in Angstroms", "1.0", 0.0);
+	pixel_size = my_input->GetFloatFromUser("Pixel size of reconstruction (A)", "Pixel size of input images in Angstroms", "1.0", 0.0);
```

Comment:
Weren't these changes already made in my last PR? Sorry, it was a large one, but does this mean your working branch is not up to date with the master?

https://github.com/timothygrant80/cisTEM/commit/0aabb63c9cc6197150891606e85719f4ea9f57b7#

Reply:
hmm I thought I had done this - and I think it would not allow automatic merge if there was a conflict.  I also moved this line, so maybe that's where it went wrong?

Reply:
Yeah, sounds like it is a semantic diff problem. Let's just make a note and double check the result in master after you merge in the PR. Maybe it is an issue with the PR diff only? I'm not sure if there is any difference in algo's used.